tests/expectations/cfg_both.c

```diff
@@ -77,6 +77,18 @@ typedef struct ConditionalField {
 #endif
   ;
 } ConditionalField;
+#define ConditionalField_ZERO (ConditionalField){ .field = __ConditionalField_field(0) }
+#if defined(X11)
+#define __ConditionalField_field(v) (v)
+#else
+#define __ConditionalField_field(v)
+#endif
+#define ConditionalField_ONE (ConditionalField){ .field = __ConditionalField_field(1) }
```

Comment:
Does this even build when it's not defined? Doesn't it expand to `.field =`?

Reply:
Thanks, fixed it